diffusion2d/output.py

```diff
@@ -0,0 +1,23 @@
+import matplotlib.pyplot as plt
+
+def create_plot(fig, fig_counter, n, u, T_cold=300, T_hot=700):
+    ax = fig.add_subplot(220 + fig_counter)
+    im = ax.imshow(u, cmap=plt.get_cmap('hot'), vmin=T_cold, vmax=T_hot)  # image for color bar axes
+    ax.set_axis_off()
+    ax.set_title('{:.1f} ms'.format(n * 1000))
+    return im
+
+def output_plots(time_list, u_list, T_cold=300, T_hot=700):
+    fig_counter = 0
+    fig = plt.figure()
+
+    for n, u in zip(time_list, u_list):
+        fig_counter += 1
+        im = create_plot(fig, fig_counter, n, u, T_cold, T_hot)
```

Comment:
It was expected that the `create_plot` function is called in `diffusion2d.py`. But what you have done is also correct.